spec/requests/custom_resources_spec.rb

```diff
@@ -235,6 +316,16 @@
                   "endCoverage": '2004-01-01'
                 }
               ],
+              "contributors": [
```

Comment:
Looks great Sobha! - just one very minor thing - should we check for contributors in response body in this test as well?